.husky/pre-commit

```diff
@@ -3,3 +3,5 @@
 
 yarn test
 yarn lint:fix
+yarn format
+git add -A
```

Comment:
this doesn't have to be in the hook
`git add -A`.
It's optional and is handled by git